applications/tari_app_utilities/Cargo.toml

```diff
@@ -22,3 +23,4 @@ thiserror = "^1.0.26"
 
 [build-dependencies]
 tari_common = { path = "../../common", features = ["build", "static-application-info"] }
+tari_features = { version = "0.0.1", path = "../../common/tari_features"}
```

Comment:
is there a reason for including it twice with the same feature set?

Reply:
I was thinking it wasn't available in the build deps unless specifically defined, although I'll double check it's needed.